src/mpi/coll/include/coll_types.h

```diff
@@ -52,6 +52,17 @@ enum {
 typedef struct MPII_Coll_req_t {
     void *sched;                /* pointer to the schedule */
 
+    /*
+     * Fields used GPU-aware fallback path for op collectives. GPU buffers
```

Comment:
This section needs some grammar fixes, the but the overall point is made. I'll wait until after the tests to modify it.

Reply:
Done.